lighthouserc.yml

```diff
@@ -1,7 +1,30 @@
 ci:
   collect:
     numberOfRuns: 1
+    startServerCommand: sh ./lighthouse.sh
+    startServerReadyPattern: 'BUILD SUCCESSFUL'
+    startServerReadyTimeout: 600000
+    url:
+      - https://localhost.simplereport.gov/app
+      - https://localhost.simplereport.gov/app?facility=$FACILITY_ID
+      - https://localhost.simplereport.gov/app/queue?facility=$FACILITY_ID
+      - https://localhost.simplereport.gov/app/results/1?facility=$FACILITY_ID
+      - https://localhost.simplereport.gov/app/patients?facility=$FACILITY_ID
+      - https://localhost.simplereport.gov/app/add-patient?facility=$FACILITY_ID
+      - https://localhost.simplereport.gov/app/settings?facility=$FACILITY_ID
+      - https://localhost.simplereport.gov/app/settings/facilities?facility=$FACILITY_ID
+      - https://localhost.simplereport.gov/app/settings/add-facility/?facility=$FACILITY_ID
```

Comment:
So if we add a new route to something, we should probably add it to this list, right?

Reply:
Yes!

Reply:
Did we want to add sign up pages, self registration, and pxp pages? I know that means creating a patient link, test order, etc... and also getting the patient link id from the DB... 😅 Not sure how much we want lighthouse to cover all parts of the app or just these pages to start 🤔

Reply:
@emyl3 I would say not right now in this ticket, I can create a new ticket to add additional pages. The complication with that is that we need better seed data for patients, which we don't have right now, and I think that'd be out of scope. @emmastephenson would you like a new ticket for testing the pages Elisa is suggesting?

Reply:
I would hold off on testing those pages through Lighthouse actually - we have e2e tests for all those flows (or we should, at least) so I think they'll be covered by Cypress axe testing.